browser/ui/webui/brave_rewards/rewards_panel_ui.cc

```diff
@@ -0,0 +1,266 @@
+    {"onboardingSetupContributeSubheader",
+     IDS_BRAVE_REWARDS_ONBOARDING_SETUP_CONTRIBUTE_SUBHEADER},
+    {"onboardingStartUsingRewards",
+     IDS_BRAVE_REWARDS_ONBOARDING_START_USING_REWARDS},
+    {"onboardingTakeTour", IDS_BRAVE_REWARDS_ONBOARDING_TAKE_TOUR},
+    {"onboardingTerms", IDS_BRAVE_REWARDS_ONBOARDING_TERMS},
+    {"onboardingTourBack", IDS_BRAVE_REWARDS_ONBOARDING_TOUR_BACK},
+    {"onboardingTourBegin", IDS_BRAVE_REWARDS_ONBOARDING_TOUR_BEGIN},
+    {"onboardingTourContinue", IDS_BRAVE_REWARDS_ONBOARDING_TOUR_CONTINUE},
+    {"onboardingTourDone", IDS_BRAVE_REWARDS_ONBOARDING_TOUR_DONE},
+    {"onboardingTourSkip", IDS_BRAVE_REWARDS_ONBOARDING_TOUR_SKIP},
+    {"onboardingTourSkipForNow",
+     IDS_BRAVE_REWARDS_ONBOARDING_TOUR_SKIP_FOR_NOW},
+    {"pendingTipText", IDS_REWARDS_PANEL_PENDING_TIP_TEXT},
+    {"pendingTipTitle", IDS_REWARDS_PANEL_PENDING_TIP_TITLE},
+    {"pendingTipTitleRegistered",
+     IDS_REWARDS_PANEL_PENDING_TIP_TITLE_REGISTERED},
+    {"platformPublisherTitle", IDS_REWARDS_PANEL_PLATFORM_PUBLISHER_TITLE},
+    {"refreshStatus", IDS_REWARDS_PANEL_REFRESH_STATUS},
+    {"rewardsLogInToSeeBalance", IDS_REWARDS_LOG_IN_TO_SEE_BALANCE},
+    {"rewardsPaymentCheckStatus", IDS_REWARDS_PAYMENT_CHECK_STATUS},
+    {"rewardsPaymentCompleted", IDS_REWARDS_PAYMENT_COMPLETED},
+    {"rewardsPaymentPending", IDS_REWARDS_PAYMENT_PENDING},
+    {"rewardsPaymentProcessing", IDS_REWARDS_PAYMENT_PROCESSING},
+    {"sendTip", IDS_REWARDS_PANEL_SEND_TIP},
+    {"set", IDS_REWARDS_PANEL_SET},
+    {"summary", IDS_REWARDS_PANEL_SUMMARY},
+    {"tip", IDS_REWARDS_PANEL_TIP},
+    {"unverifiedCreator", IDS_REWARDS_PANEL_UNVERIFIED_CREATOR},
+    {"verifiedCreator", IDS_REWARDS_PANEL_VERIFIED_CREATOR},
+    {"walletAccountLink", IDS_REWARDS_WALLET_ACCOUNT_LINK},
+    {"walletAddFunds", IDS_REWARDS_WALLET_ADD_FUNDS},
+    {"walletAutoContribute", IDS_REWARDS_WALLET_AUTO_CONTRIBUTE},
+    {"walletDisconnected", IDS_REWARDS_WALLET_DISCONNECTED},
+    {"walletDisconnectLink", IDS_REWARDS_WALLET_DISCONNECT_LINK},
+    {"walletEstimatedEarnings", IDS_REWARDS_WALLET_ESTIMATED_EARNINGS},
+    {"walletLogIntoYourAccount", IDS_REWARDS_WALLET_LOG_INTO_YOUR_ACCOUNT},
+    {"walletMonthlyTips", IDS_REWARDS_WALLET_MONTHLY_TIPS},
+    {"walletOneTimeTips", IDS_REWARDS_WALLET_ONE_TIME_TIPS},
+    {"walletPending", IDS_REWARDS_WALLET_PENDING},
+    {"walletPendingText", IDS_REWARDS_WALLET_PENDING_TEXT},
+    {"walletRewardsFromAds", IDS_REWARDS_WALLET_REWARDS_FROM_ADS},
+    {"walletRewardsSummary", IDS_REWARDS_WALLET_REWARDS_SUMMARY},
+    {"walletUnverified", IDS_REWARDS_WALLET_UNVERIFIED},
+    {"walletVerified", IDS_REWARDS_WALLET_VERIFIED},
+    {"walletYourBalance", IDS_REWARDS_WALLET_YOUR_BALANCE}};
+
+}  // namespace
+
+RewardsPanelUI::RewardsPanelUI(content::WebUI* web_ui)
+    : MojoBubbleWebUIController(web_ui, true) {
+  auto* profile = Profile::FromWebUI(web_ui);
+  if (auto* browser = chrome::FindLastActiveWithProfile(profile)) {
```

Comment:
you should get the browser from the web_ui->GetWebContents

Reply:
Unfortunately, it turns out that `FindBrowserWithWebContents(web_ui->GetWebContents())` returns `nullptr` in this case.

Reply:
oh right, because that is the webui webcontents